### PR TITLE
Simple GitHub action to ensure release note entry included in PRs into main.

### PR DESCRIPTION
**Pull request by derekpierre**
The check that this PR adds, is expected to fail for this PR hehehe 😅 .

Release note entry format readme can be viewed - https://github.com/derekpierre/nucypher/blob/release-notes-action/newsfragments/README.md.

### CODE REVIEW

**Comment by derekpierre** [comment]
Side note: with this change we don't need a #github-alerts failure message in Discord if the PR release note entry is not found. We can probably tone down the events for the Discord webhook (currently set to all events).

**Review by KPrasch — COMMENTED** [review]
How about a release note for this PR?

File: .github/workflows/pr_release_note_entry.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      - '*.*'  # ignore releases
+
+jobs:
+  release-note-entry:
+    name: 'Checking release note entry for PR ${{ github.event.number }}'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          test -f './newsframents/${{ github.event.number }}.*'
```
Comment (fjarri) [inline .github/workflows/pr_release_note_entry.yml:17]: `newsfragments`?

Comment (derekpierre) [reply .github/workflows/pr_release_note_entry.yml]: ✔️ - whoops thanks! 🤦

**Comment by derekpierre** [comment]
> How about a release note for this PR?
@KPrasch feels like this PR is an internal thing, and not really applicable to release notes - unless you are thinking for open source contributors who make PRs into our repos.

**Comment by derekpierre** [comment]
@KPrasch Added a release note entry (helps to test anyway).

**Comment by derekpierre** [comment]
Moving back to WIP. Fiddling with a few things.

**Comment by derekpierre** [comment]
All good now - ready for review. Quotes in the `test` command caused regex issue.

**Comment by derekpierre** [comment]
Really great tool for testing github actions locally - https://github.com/nektos/act.

**Review by KPrasch — APPROVED** [review]
:cowboy_hat_face: - Excellent!

**Review by vzotova — APPROVED** [review]

**Review by cygnusv — APPROVED** [review]

**Review by tuxxy — APPROVED** [review]

**Pull request merged**